samples/bluetooth/peripheral_with_multiple_identities/README.rst

```diff
@@ -0,0 +1,253 @@
+==================================
+
+Each advertiser is set up as a **connectable legacy advertiser** with a long advertisering interval.
+These values can be tailored to specific use cases through the structure :c:struct:`bt_le_adv_param`.
+* Set the :c:member:`bt_le_adv_param.id` field to assign a given identity to an advertiser.
+
+The extended advertising version of the API is used as it supports multiple advertising instances and sets.
+These instances are stored in an array where the advertisers identity serves as the index, to simplify the retrieval and management.  
+A unique name is generated for each advertiser combining :ref:`CONFIG_BT_DEVICE_NAME <CONFIG_BT_DEVICE_NAME>` with its identity and this is set by the :c:func:`bt_le_ext_adv_set_data` API.
+
+Starting an advertiser
+======================
+
+A work item is then initialized for each advertiser to ensure tasks are executed independently, preventing race conditions and avoiding blocking within callback functions.
+Each advertiser's work item can then be submitted to the work queue, where it is started using the :c:func:`bt_le_ext_adv_start()` API. 
+By default, this API does not attempt to resume connectable advertising after a connection is established. 
+
+Connection callbacks
+====================
+
+After the initial set up of each advertiser, the sample continues to execute asynchronously using callbacks to define its behavior.
+The sample uses the following two connection callbacks (see the :c:struct:`bt_conn_cb` structure):
+
+* **Connected callback:** When a connection is established the sample prints the central's address. 
+* **Disconnected callback:** When a disconnection occurs, the central's address is printed along with the advertiser that was disconnected.
+   The identity associated with the connection is retrieved by calling :c:func:`bt_conn_get_info()` to obtain :c:member:`bt_conn_info.id`.
+   The work item (advertiser) associated with this identity is then submitted to the work queue to restart the advertiser.
+   The identity is used as the index to retrieve the associated advertiser from the array. 
+
+Extending the sample
+====================
+
+To get more out of this sample you can add some features as listed below:
+
+#. **Increase the number of advertisers:** Modify the configuration options listed above to support more advertisers.
+#. **Customise individual advertisers:** :c:struct:`advertiser_info` can be extended to customise advertising parameters, advertising data and name for each advertiser.
+#. **Custom advertising data:** The advertising data can easily be customised to include additional information, in addition to supporting scan responses.
+#. **Reuse identities for multiple advertisers:** Once all identities have been initialized the sample supports reuse for other advertisers.
+#. **Advanced features:** The sample can be extended to implement other Bluetooth features or used in conjunction with other samples.
```

Comment:
```suggestion
* Increase the number of advertisers - Modify the configuration options listed above to support more advertisers.
* Customize individual advertisers - :c:struct:`advertiser_info` can be extended to customize advertising parameters, advertising data, and a name for each advertiser.
* Custom advertising data - The advertising data can easily be customized to include additional information, in addition to supporting scan responses.
* Reuse identities for multiple advertisers - Once all identities have been initialized, the sample supports reuse for other advertisers.
* Advanced features - The sample can be extended to implement other Bluetooth features or used in conjunction with other samples.
```
I think we could avoid using bold here as well. Could you confirm this @peknis .